api/lib/infrastructure/repositories/student-repository.js

```diff
@@ -26,5 +26,19 @@ module.exports = {
   batchSave(studentsToSave) {
     const bookshelfStudents = studentsToSave.map((studentToSave) => _.omit(studentToSave, ['id']));
     return Bookshelf.knex.batchInsert('students', bookshelfStudents).then(() => undefined);
+  },
+
+  checkIfUserIsPartOfStudentListInOrganization({ user, organizationId }) {
+    return BookshelfStudent
+      .query((qb) => {
+        qb.where('organizationId', organizationId);
+        qb.whereRaw('? in (LOWER(??), LOWER(??), LOWER(??))', [user.firstName.toLowerCase(), 'firstName', 'middleName', 'thirdName']);
```

Comment:
Attention `.toLowerCase()` peut ne pas donner le même résultat que la fonction SQL `LOWER()`. Il serait plus prudent à mon sens de faire :
```suggestion
        qb.whereRaw('LOWER(?) in (LOWER(??), LOWER(??), LOWER(??))', [user.firstName, 'firstName', 'middleName', 'thirdName']);
```

Reply:
Ça marche, je l'ajoute !